include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -154,6 +181,14 @@ struct CShuffleEpilogue
                                               tile_distribution_pattern::thread_raked>;
         constexpr auto dram_tile_distribution = TileEncodingPattern::Make2DStaticTileDistribution();
 
+        auto d_dram_small_window = generate_tuple(
+            [&](auto idx) { return make_tile_window(ds_dram_window[idx], dram_tile_distribution); },
+            number<NumDTensor>{});
+
+        using elemenet_wise_output_t =
+            decltype(load_tile(make_tile_window(out_lds_window, dram_tile_distribution)));
+        elemenet_wise_output_t elemenet_wise_output;
```

Comment:
Why this is actually needed? Why not just overwrite `c_out_tensor` - this would use much less registers